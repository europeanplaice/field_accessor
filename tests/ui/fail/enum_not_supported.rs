use field_accessor::FieldAccessor;

#[derive(FieldAccessor)]
enum Example {
    Variant,
}

fn main() {}
