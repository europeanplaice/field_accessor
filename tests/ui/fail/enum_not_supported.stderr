error: proc-macro derive panicked
 --> tests/ui/fail/enum_not_supported.rs:3:10
  |
3 | #[derive(FieldAccessor)]
  |          ^^^^^^^^^^^^^
  |
  = help: message: Enum is not supported. Only struct is supported
